addons/medical/CfgEventHandlers.hpp

```diff
@@ -33,7 +33,7 @@ class Extended_Init_EventHandlers {
 class Extended_Respawn_EventHandlers {
     class CAManBase {
         class ADDON {
-            respawn = QUOTE(call COMPILE_FILE(XEH_respawn));
+            respawn = QUOTE(call FUNC(handleRespawn));
```

Comment:
CBA_fnc_addClassEventHandler both in preInit.
This would've avoided this problem in the first place.

Reply:
Should I also use addCEH for displayLoad?